front/app/containers/Admin/reporting/components/ReportBuilder/TopBar/index.tsx

```diff
@@ -212,19 +204,37 @@ const ContentBuilderTopBar = ({
             </Text>
           )}
         </Box>
-        <LocaleSwitcher
-          selectedLocale={selectedLocale}
-          onSelectLocale={setSelectedLocale}
-        />
-        <Box mx="20px">
-          <PrintReportButton reportId={reportId} />
+        <Box>
+          <LocaleSelect locale={selectedLocale} setLocale={setSelectedLocale} />
```

Comment:
Maybe I missed this discussion, but why do we need to make it different from other content builders?